flink-datastream-api/src/main/java/org/apache/flink/datastream/api/context/RuntimeContext.java

```diff
@@ -41,6 +42,9 @@ public interface RuntimeContext {
     /** Get the {@link ProcessingTimeManager} of this process function. */
     ProcessingTimeManager getProcessingTimeManager();
 
+    /** Get the metric group of this process function. */
+    OperatorMetricGroup getMetricGroup();
```

Comment:
Why would we expose the entire OperatorMetricGroup? It might be good enough to expose a MetricGroup that allow users to define custom metrics in a process function.